src/ia-book-actions.js

```diff
@@ -129,10 +128,19 @@ export default class IABookActions extends LitElement {
     // clear all intervals for lending system
     window?.IALendingIntervals?.clearAll();
 
-    window?.Sentry?.captureMessage(sentryLogs.disconnectedCallback);
+    this.sentryCaptureMsg(sentryLogs.disconnectedCallback);
     this.disconnectResizeObserver();
   }
 
+  /**
+   * send log messages to sentry
+   * @param {string} msg
+   */
+  sentryCaptureMsg(msg) {
+    log(window?.Sentry);
+    if (window?.Sentry) window?.Sentry?.captureMessage(msg);
```

Comment:
q: do we need conditional if statement has optional chain? can we remove conditional?